### DISCUSSION TITLE
［补档］B站上的一些好东西(≖ ◡ ≖✿)

### DISCUSSION THREAD

**Discussion by fzdqz** [share121/inter-knot#4789] [灌水]
美好知识的传送入口，懂得都懂，不用谢我，我是个好人 ヾ(❀╹◡╹)ﾉ~
https://www.bilibili.com/read/readlist/rl330029
 https://www.bilibili.com/read/readlist/rl331197
 https://www.bilibili.com/read/readlist/rl334156
 https://www.bilibili.com/read/readlist/rl336027
 https://www.bilibili.com/read/readlist/rl336028

**Comment by RaInWH** [comment]
确实是好东西啊

**Comment by lllliufeng** [comment]
我就知道

**Reply by fzdqz** [reply]
起码不是rick摇:)

**Comment by HyperMuteki0347** [comment]
这是真的好东西啊有时间好好学习一下<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8481df2cd99d2b00ecb94a92a14c9fe4_7915718047845434249.png" alt="米游兔 OK" width="75" >

**Reply by fzdqz** [reply]
青年大学习<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by BJ-cmyk** [comment]
哥哥哥，之前的vpn链接能补档吗？😭

**Reply by fzdqz** [reply]
https://play.google.com/store/apps/details?id=com.ambrose.overwall 这是之前那个VPN的|•'-'•)و✧

**Reply by BJ-cmyk** [reply]
谢谢嗷

**Comment by aoto537** [comment]
最后一个怎么打不开

**Reply by fzdqz** [reply]
被删了😔
